dragon_gigapet/Dragon.rb

```diff
@@ -0,0 +1,39 @@
+class Dragon < Animal  # Do I still need to add instant variables to initalize if already in animal file 
+    
+
+
+public
```

Comment:
You shouldn't need to explicitly declare public methods. Methods are public by default.

Reply:
So only private methods need to be declared.